### PR TITLE
Keras Cifar10 MLP

### PR DESCRIPTION
**Pull request by ryonakai**
- Accuracy ~33 %

### CODE REVIEW

File: nakai/cifar10_mlp.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+from keras.layers.core import Dense, Dropout, Activation, Flatten
+from keras.layers.convolutional import Convolution2D, MaxPooling2D
+from keras.optimizers import SGD
+from keras.utils import np_utils
+
+batch_size = 32
+nb_classes = 10
+nb_epoch = 10000
+data_augmentation = True
+
```
Comment (ryonakai) [inline nakai/cifar10_mlp.py:27]: data augumentation はしていない

File: nakai/cifar10_mlp.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+model.add(Dense(512, input_shape=(32*32*3,)))
+model.add(Activation('relu'))
+model.add(Dropout(0.5))
+model.add(Dense(nb_classes))
+model.add(Activation('softmax'))
+
+# let's train the model using SGD + momentum (how original).
+sgd = SGD(lr=0.01, decay=1e-6, momentum=0.9, nesterov=True)
+model.compile(loss='categorical_crossentropy',
```
Comment (ryonakai) [inline nakai/cifar10_mlp.py:82]: learning rate　について
